### PR TITLE
std.path.toNormalizedPath - fix typo

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

**Comment by schuetzm** [comment]
Can you add a unittest that tests this path? Apparently there is none, because otherwise this wouldn't even have compiled.

File: std/path.d [hunk 1]
```diff
@@ -1787,7 +1787,7 @@ auto toNormalizedPath(R)(R path)
             }
             else
             {
-                C = element.front;
+                C c = element.front;
```
Comment (andralex) [inline std/path.d]: how did this pass?

**Comment by WalterBright** [comment]
This is just to get some other work I'm doing in Phobos moving. I'll revisit the unit test coverage later, which is clearly inadequate.
